### DISCUSSION TITLE
HIP-00XX: Establish a Multisig for Handshake.org Domain Transfer, Tokenization, and Brand Control

### DISCUSSION THREAD

**Discussion by pcfreak30** [handshake-org/HIPs#70] [Polls]
# HIP-00XX: Establish a Multisig for Handshake.org Domain Transfer, Tokenization, and Brand Control

## Overview

This proposal aims to establish a trusted multisig for transferring the domain rights of handshake.org, subsequently tokenizing it, and managing the Handshake brand assets, including the Twitter account. This action is crucial for ensuring the decentralized governance and longevity of the Handshake project's primary domain and brand presence.

_The current holder of `handshake.org` is Andrew Lee (of purse.io)_, a HNS founder

## Background

Handshake.org is the central hub for information about the Handshake project, and its associated brand assets are vital for community engagement and project visibility. Properly managing these assets is critical for the project's continuity and community trust.

### 1. Multisig Structure

We propose an 11-person multisig structure for controlling the handshake.org domain and associated brand assets. The suggested threshold is 9 out of 11 signatures, but we welcome community input on alternative thresholds. The multisig will be implemented using a Gnosis Safe, a battle-tested Ethereum multisig wallet.

### 2. Proposed Multisig Participants

Based on their long-standing contributions to the Handshake ecosystem, technical expertise, and community trust, we propose the following individuals for the multisig:

1. NetOpWibby
2. eskimo
3. Michelini
4. nathanwoodburn
5. crymzyn
6. nodech
7. rithvikvibhu
8. pcfreak30
9. falci
10. Durendil
11. pinheadmz

Note: This list is open for community discussion and modification.

### 3. Responsibilities of Multisig Participants

- Execute the will of the community regarding the handshake.org domain and brand assets
- Ensure the domain remains active and properly configured
- Facilitate the tokenization process of the domain
- Manage the Handshake Twitter account
- Coordinate with any future governance or funding bodies established by the community
- Take action in consensus if there is a technical problem or security threat to the assets

### 4. Domain Management and Security

- We will use a Cloudflare account for managing the handshake.org domain
- The multisig members will have the authority to take immediate action, in consensus, if there is a technical problem or security threat to any of the managed assets

### 5. Domain Renewal, Tokenization, and Brand Asset Management

- Fund the domain renewal and tokenization costs for 10 years:
  - Domain renewal: $10/year
  - Tokenization fee: $6/year
  - Total for 10 years: (10 + 6) * 10 = $160
- Tokenize the domain on the Optimism network
- Transfer control of the Handshake Twitter account to an email address @handshake.org
- Set up a lifetime email account for $150

### 6. Email Service

- Establish a lifetime email service for the domain at $150
- This email will be used for managing the Twitter account and other brand-related communications

### 7. Total Funding Needed

The total funding required for this proposal is:
- Domain renewal and tokenization for 10 years: $160
- Lifetime email account: $150
- Total: $310

## Implementation Steps

1. Community Discussion: Open this proposal for community feedback and suggestions
2. Finalize Multisig Participants: Reach consensus on the 11 individuals who will control the multisig
3. Determine Multisig Threshold: Agree on the number of required signatures (e.g., 9/11 or an alternative suggested by the community)
4. Technical Setup: Establish the Gnosis Safe multisig wallet with the agreed-upon threshold
5. Funding: Initiate the funding process for the total amount of $310
6. Domain Transfer: Transfer handshake.org to the multisig wallet and set up Cloudflare management
7. Tokenization: Complete the tokenization process on Optimism
8. Email Setup: Establish the lifetime email service for the domain
9. Twitter Transfer: Transfer the Handshake Twitter account to the new @handshake.org email

## Relationship to Wider Governance

This multisig structure is designed to work in harmony with any future governance or funding bodies that may be established for the Handshake project. The multisig participants will act as executors of community decisions, not as sole decision-makers, except in cases of immediate technical or security threats.

## Next Steps

1. Open this proposal for community discussion on GitHub
2. Gather feedback and suggestions over a 2-week period
3. Revise the proposal based on community input
4. Hold a final vote to approve the multisig structure, participants, and threshold

We encourage all members of the Handshake community to provide their input on this crucial decision for the project's future, including suggestions for the multisig threshold and any other aspects of this proposal.

**Comment by Nathanwoodburn** [comment]
Can you provide info on the tokenization process and the lifetime email?

**Reply by pcfreak30** [reply]
Tokenization uses https://3dns.box which is what powers the `.box` tld, I own one. its optimism powered and will use a gnosis safe. They also confirm direct integration with Safe.

email will be https://mxroute.com unless anyone has a better option. I have used them for a while and its a great no-thrills email service.
